### PR TITLE
fix(data validator): list of unique view names with same elements order

### PR DESCRIPTION
**Pull request by juliayakovlev**
longevity-lwt-500G-3d-test failed with error:
```
AlreadyExists: Table 'cqlstress_lwt_example.blogposts_update_one_column_lwt_indicator_expect' already exists
```

It happened because of test-cases/longevity/longevity-lwt-500G-3d.yaml runs the same user profile a few times, so list of view names is not unique, like:
```
['blogposts_update_one_column_lwt_indicator', 'blogposts_update_2_columns_lwt_indicator', 'blogposts_update_one_column_lwt_indicator', 'blogposts_update_2_columns_lwt_indicator', 'blogposts_update_one_column_lwt_indicator', 'blogposts_update_2_columns_lwt_indicator']
```

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by fruch** [comment]
@juliayakovlev what about backports ? is this part of the initial fix  or part of the extra debugging information ?

File: sdcm/utils/data_validator.py [hunk 1]
```diff
@@ -309,7 +309,7 @@ def get_view_name_from_profile(self, name_substr, cs_profile=None, all_entries=F
                     if not all_entries:
                         break
 
-        return mv_names
+        return list(dict.fromkeys(mv_names))  # list of unique view names and keep the elements order
```
Comment (fruch) [inline sdcm/utils/data_validator.py:312]: why are you using `dict.fromkeys` and not `set` ? does it behave any differently ?

Comment (juliayakovlev) [reply sdcm/utils/data_validator.py:312]: `set` does not keep the element order (this was the original problem, that caused the failure). `dict.fromkeys` keeps the order

**Review by fruch — APPROVED** [review]
LGTM

**Pull request merged**